core/sail/lmdb/src/main/java/org/eclipse/rdf4j/sail/lmdb/ContextStore.java

```diff
@@ -0,0 +1,230 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.sail.lmdb;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.EOFException;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Objects;
+
+import org.eclipse.rdf4j.common.io.IOUtil;
+import org.eclipse.rdf4j.common.iteration.CloseableIteration;
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.Resource;
+import org.eclipse.rdf4j.model.ValueFactory;
+import org.eclipse.rdf4j.sail.SailException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * An in-memory index for context information that uses a file for persistence.
+ * <p>
+ * The context index file has an 8-byte header consisting of:
+ *
+ * <pre>
+ * 	byte 1-3         : the magic number marker
+ *  byte 4           : the file format version
+ *  byte 5-8         : the number of mapped contexts contained in the file, as an int.
+ * </pre>
+ *
+ * Each context is encoded in the file as a record, as follows:
+ *
+ * <pre>
+ *   byte 1 - 8      : the number of statements in the content, as a long.
+ *   byte 9          : boolean flag indicating the type of context identifier (1 = IRI, 0 = blank node)
+ *   byte 10 - 11    : the length of the encoded context identifier
+ *   byte 12 - A     : the UTF-8 encoded the encoded context identifer
+ * </pre>
+ *
+ * @author Jeen Broekstra
```

Comment:
I think this Author field should be different.

Reply:
Some files like ContextStore.java are just (almost) verbatim copies from NativeStore. Therefore I didn't touch the author fields.
Should I change or extend the field even for small changes?